docs/configuration-reference/components/contour.md

```diff
@@ -68,13 +66,14 @@ Table of all the arguments accepted by the component.
 Example:
 
 | Argument            | Description                                                                                             | Default        | Required |
-|---------------------|---------------------------------------------------------------------------------------------------------|:--------------:|:--------:|
+|---------------------|---------------------------------------------------------------------------------------------------------|:----------------:|:----------:|
```

Comment:
The width of last column should remain the same I think.

Reply:
Updated